src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const activeTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const isCurrentTabActive = activeTab.id === tab.id;
+
+            return (
+              <li
+                key={tab.id}
+                data-cy="Tab"
+                className={classNames({ 'is-active': isCurrentTabActive })}
+              >
+
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (selectedTabId !== tab.id) {
+                      onTabSelected(tab);
+                    }
+                  }}
```

Comment:
it is bad practise to make anonymous arrow function in jsx
create handler function in component and pass just reference here
```
const selectTab = () => {
  if (selectedTabId !== tab.id) {
    onTabSelected(tab);
  }
}
```
```suggestion
                  onClick={selectTab}
```